MentalHospital.Tests/Tests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Xunit;
+using Moq;
+using AutoMapper;
+using Shouldly;
+using MentalHospital.BLL.Mapper;
+using MentalHospital.BLL.Models;
+using MentalHospital.BLL.Services;
+using MentalHospital.DAL.Interfaces;
+using MentalHospital.DAL.Entities;
+
+
+namespace MentalHospital.Tests
+{
+    public class Tests
```

Comment:
Rename this file to be able to understand which file we are testing

Reply:
And the same for project
Name of project + Tests - for project name
Name of file + Tests - for file name